frontend/src/components/forms/CreateLog.tsx

```diff
@@ -283,18 +283,33 @@ const CreateLog = ({
     // Create a log in the db with this data
     setCreateOpen(false);
     // update the table with the new log
-    LogRecordAPIClient.createLog(employee.value, resident, flagged, notes, attnTo, building).then((res) => {
-      if (res != null) {
-        setAlertData(ALERT_DATA.SUCCESS)
-        countRecords()
-        getRecords(1);
-        setUserPageNum(1)
-      }
-      else {
-        setAlertData(ALERT_DATA.ERROR)
-      }
-      setShowAlert(true);
-    })
+    if (attnTo !== -1) {
```

Comment:
nit: I forgot to do this before, but it might be be helpful to just mark what `attnTo` represents when its -1

Reply:
Unless I didn't forget to do this, and we already commented it somewhere lol

Reply:
Is a comment specifying that -1 is the default value for `attnTo` what you would be looking for?